let myList = []

try{
    console.log(myList.x.test)
}
catch(e){
    console.log('متاسفانه خطایی رخ داده است.');
    console.log(e);
}
console.log('salam');